packages/apollo-link-error/src/__tests__/index.ts

```diff
@@ -232,6 +233,68 @@ describe('error handling', () => {
       done();
     });
   });
+
+  it('allows re-forwarding in the error handler', done => {
+    const query = gql`
+      query Foo {
+        foo {
+          bar
+        }
+      }
+    `;
+
+    let called;
+    let retryCalled;
+    const errorLink = onError(
+      ({ graphQLErrors, response, operation, forward }) => {
+        if (graphQLErrors.length > 0) {
+          called = true;
+          expect(graphQLErrors[0].message).toBe('resolver blew up');
```

Comment:
Let's surround these expects in a try -> catch -> done.fail, so that if the expect fail, then we get a good message

Reply:
Fixed